core/benchmarks/DiskBlockObjectWriterCloseMethodBenchmark-results.txt

```diff
@@ -0,0 +1,48 @@
+================================================================================================
+emptyDataBuffer
```

Comment:
Although the else branch is entered only when 'objectsWritten = = 0', there still seem to be some performance differences between the `close` and `revertPartialWritesAndClose` method.

I run the bench with ssd，I don't know if there will be a more obvious difference if HDD is used